src/main.rs

```diff
@@ -509,12 +509,17 @@ fn open_report(cx: &Context, path: &Utf8Path) -> Result<()> {
 
 fn merge_profraw(cx: &Context) -> Result<()> {
     // Convert raw profile data.
+    let profraw_files =
+        glob::glob(cx.ws.target_dir.join(format!("{}-*.profraw", cx.ws.name)).as_str())?
+            .filter_map(Result::ok);
+    let mut input_files = tempfile::NamedTempFile::new()?;
+    for path in profraw_files {
+        writeln!(input_files, "{}", path.display())?;
```

Comment:
Maybe it would be more robust to assume valid UTF-8 here and fail loudly if that does not hold. I fear that `Path::display` performing a lossy translation of the paths could lead to hard-to-diagnose errors down the road.

Reply:
Done.